docs/configuration.md

```diff
@@ -2,67 +2,31 @@
 
 ### Table of Contents
 
-- [VisibilityConfigObject][1]
-  - [Parameters][2]
-  - [Examples][3]
-- [FloatingConfigObject][4]
-  - [Parameters][5]
-- [EventType][6]
-  - [PLAYER_VISIBILITY_CHANGED][7]
-  - [FLOATING_PLAYER_STATE_CHANGED][8]
-  - [FLOATING_PLAYER_DISMISSED][9]
-
-## VisibilityConfigObject
-
-Visibility plugin configuration parameters are provided whenever a player instance is created.
-
-Type: [Object][10]
-
-### Parameters
```

Comment:
why did you remove these docs?

Reply:
It was generated by script. In general I removed the visibility level of the plugin config. We have the floating directly